internal/tracing/attributes/attributes.go

```diff
@@ -0,0 +1,16 @@
+package attributes
+
+const (
+	// Common attributes names used across the codebase
+	ProviderAddress = "opentofu.provider.address"
+	ProviderVersion = "opentofu.provider.version"
+
+	ErrorSourceType = "opentofu.error.source.type"
+
+	TargetPlatform = "opentofu.targetplatform"
+
+	// Provider Locking attributes
+	ProviderLockTargetPlatforms = "opentofu.provider.lock.targetplatforms"
+	ProviderLockFSMirror        = "opentofu.provider.lock.fsmirror"
+	ProviderLockNetMirror       = "opentofu.provider.lock.netmirror"
```

Comment:
I'm a little doubtful about placing attribute names that are only used by one subsystem in this central place, since there's no need for multiple callers to agree on these ones and it'll presumably cause future system changes to all compete to change this one file, and thus increase the likelihood of conflicts.

Could we compromise and say that `package attributes` (or indeed `package traceattrs`) is focused only on the cross-cutting attribute names that we intend to use consistently throughout the codebase, and let these very specialized ones just be written directly inline for simplicity's sake?

Reply:
That works for me